fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -1551,6 +1597,9 @@ ACTOR Future<Void> commitBatch(ProxyCommitData* self,
 	}
 	self->logSystem->popTxs(msg.popTo);
 
+	state double afterTlogLogging = now();
+	self->stats.tlogLoggingDist->sampleSeconds(afterTlogLogging - finishProcessingMutation);
```

Comment:
We can log the latency to each logs inside the log push() function: 
https://github.com/apple/foundationdb/blob/149e73d19f2c4df14d824c3408e32a61a13c2858/fdbserver/TagPartitionedLogSystem.actor.cpp#L570-L576